README.md

```diff
@@ -17,3 +27,9 @@ Enable ADB USB mode in settings and push the usr folder to your root directory.
 | File | License |
 | --- | --- |
 | retro-lcd - [Digital-7 (Mono)](retro-lcd/usr/share/fonts/digital-7%20(mono).ttf) | The font "Digital-7" is freeware for home use and was created by [Sizenko Alexander](http://www.styleseven.com). |
+| binary-lcd - [Digital-7 (Mono)](binary-lcd/usr/share/fonts/digital-7%20(mono).ttf) | The font "Digital-7" is freeware for home use and was created by [Sizenko Alexander](http://www.styleseven.com). |
+| bold-hour-bebas - [BebasKai-Regular](bold-hour-bebas/usr/share/fonts/BebasKai-Regular.otf) | The font "BebasKai" is freeware for home use and was created by [Dharma Type](http://www.dharmatype.com/). [license.txt](bold-hour-bebas/usr/share/fonts/License.txt) |
+| contemporary-digital-rings - [Exo2-ExtraLight](contemporary-digital-rings/usr/share/fonts/Exo2-ExtraLight.otf), [Exo2-Regular](contemporary-digital-rings/usr/share/fonts/Exo2-Regular.otf) | The font "Exo2" is freeware for home use and was created by [Natanael Gama](http://www.ndiscovered.com/). |
+| day-clock-24h - [Vollkorn-Regular](day-clock-24h/usr/share/fonts/Vollkorn-Regular.ttf) | The font "Vollkorn" is freeware for home use and was created by [Friedrich Althausen](http://www.vollkorn-typeface.com). [license](day-clock-24h/usr/share/fonts/OFL.txt)|
```

Comment:
According to https://github.com/dharmatype/Bebas-Neue Bebas is licensed under Open Font License
According to https://fonts.google.com/specimen/Exo+2 Exo2 is licensed under Open Font License
According to https://www.fontsquirrel.com/fonts/vollkorn Vokorn is licensed under Open Font License

Reply:
BebasKai is different to BebasNeue, so i will keep the link to the original DharmaType website and license text.
Exo 2 and Vollkorn corrected.